synapse/tests/test_lib_ast.py

```diff
@@ -1351,7 +1351,7 @@ async def test_lib_ast_module(self):
                     {n.ndef for n in nodes})
 
             msgs = await core.stormlist('pkg.list')
-            self.stormIsInPrint('foo                             : 0.0.1', msgs)
+            self.stormIsInPrint(' foo                                       0.0.1', msgs)
```

Comment:
would it be worth considering using `whitespace=false` here to avoid needing to tweak the tests for future column width adjustments?

Reply:
Nice! I've added it.